### ISSUE TITLE
Very early game (Balance Challenge)

### ISSUE THREAD

**Issue by darkniko** [genbtc/AutoTrimps#29]
Hello i've been using AT for 2 days now during Balance Challenge runs and i encounter some issues.

1st of all thank u (whoever corresponds) for creating such marvelous scrypt. When the game starts at z1 it does an excellent job. It is incredible.

The issues i found are related to Balance Stacks. While stacks are on, AT says it needs "more health". I checked that problem a few times and realise that going into a very very low level map (like level 6) would solve the issue.
Its like for example the current zone deals 1T damage and with 250 stacks im at 300B Health. But if i do a map and lower those 250 stacks i would be at 2T Health ... 

So its like AT doesn't notice about the fact that im doing a Balance Challenge recieving Balance Stacks that lowers my health. IT just does the current zone and then goes to map for pretiges and farming...

Another issue i encountered is that in 30+ Zones going for more than 100% bonus damage in maps is innecesary. More than a few times i've seen that my damage almost equals the health of enemies in current zones. I think thats not what it should do. Cause being able to sustain the damage recieved and being able to hit 30-40% health of enemy is sufficient.

Also about BLK damage. Sometimes it goes for lvl 10-20 Block shield when i realised that buying only 5 levels and buying a new prestige is better.

------------
#### Is there a way to make At not to farm 10 times the maps. Instead making it farm only 5 times?
------------

I tried Selecting Balance option in Core Tab but couldn't notice any significant difference. Maybe im doing something wrong., i really don't know. Most of the "high level" users i asked about AT told me that this scrypt wasn't intended for very very early game and might not be adecuate.. Still wanted to test it.

Overall AT made my game experience way more simplier and fast.

**Comment by LucidCrux** [comment]
If you are going to run a map 5 times, you might as well do it 10. By running it 5 you wipe out all stacks you'd get from the zone anyways. This is definitely not set up for Balance runs. The Balance you selected was for selecting the challenge to run on auto portal. There is no Balance challenge setting. I'm about where you are.

I'm not entirely sure that running it normal is not a good enough way. I was thinking of running a test in two windows, one with automap one without. If it is better to keep stacks instead of worrying about zone bonus, then the ideal way is to only run maps in order to get items and then leave. That way, after 5 zones, you are always back up near max stacks at the end of every world because you come out a net positive of 25 or 50 stacks. Even if you stall out for a while on the world map, you would be making 350% resources.

So basically, a way to switch automap mode to force prestige mode (I think) only would fit that. And then once you get to the level you want to do void maps, it just idles until you're ready. Maybe eventually running maps to get rid of all the stacks right before running the void maps.

**Comment by darkniko** [comment]
Im not doing void maps at all. Just go for the 2k He run and portal inmediately. I tend to think that void maps are a distraction in Balance challenge cause the purpose of it is to do Helium not to farm for getting prestiges and good amounts of damage.

Im just trying to know if there is a way to "configure" my AT so that when i has 100 stacks it goes directly to a very low level of map so the prompt of "need more Health" dissapears and jump directly to zone again... Besides why would i want to waste time in getting 100% more damage doing 5 more runs if 100% is more than enough to do 30-40% of the enemy's health. Its not that i 1 hit ko in maps it takes too much time away... But nvm seems ill have to learn scrypting so i can edit the scrypt myself 

#### For example between zones 6 - 11 farming prestiges/health/damage is extremely unecesary cause AT buy so much damage that u end up having 100k at zone 6 and until z11 u can't pass it cause of your blockdmg. But still u keep 1 hittin all enemies until zone 13 .. Which is hilarious why At keeps map farming while it could do it from 13 onwards .. But hey .. its ok thnx anyway to answer :)
